primefaces/src/main/resources/META-INF/resources/primefaces/core/core.ajax.js

```diff
@@ -86,6 +86,12 @@ if (!PrimeFaces.ajax) {
             'onco': 'oncomplete'
         },
 
+        /**
+         * Minimum number of milliseconds to show Ajax load animations.
+         * @type {number}
+         */
+        minLoadAnim : 500,
```

Comment:
lets use "Animation" instead of "Anim", no need to shorten it actually
min/max is common

Reply:
Will do.